makeflow/src/makeflow.c

```diff
@@ -1891,17 +1980,19 @@ int main(int argc, char *argv[])
 
 	makeflow_hook_create(hook_args);
 
-	if((argc - optind) != 1) {
+	if((argc - optind) == 1) {
+		if (dagfile)
+			fatal("only one dagfile can be specified");
```

Comment:
Why?

Reply:
If multiple dagfiles are specified, I'd rather not choose by some unspecified ordering, and just error out instead.